dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ExclusionFileParser.java

```diff
@@ -69,13 +70,29 @@
   static ImmutableList<LinkageErrorMatcher> parse(Path exclusionFile)
       throws SAXException, IOException, VerifierConfigurationException {
 
+    InputSource inputSource = new InputSource(Files.newInputStream(exclusionFile));
+    inputSource.setSystemId(exclusionFile.toUri().toString());
+
+    return parse(inputSource);
+  }
+
+  static ImmutableList<LinkageErrorMatcher> parse(URL exclusionFile)
```

Comment:
`linkage-checker-exclusion-default.xml` is a file in a JAR file when running as enforcer rule. So picking up URL rather than Path.